cg/meta/report/api.py

```diff
@@ -177,6 +185,7 @@ def _fetch_family_samples_from_status_db(self, family_id: str) -> list:
 
         for family_sample in family_samples:
             sample = family_sample.sample
+            self._check_sample_statuses(sample)
```

Comment:
I have a suggestion on this when we have some time to talk about it

Reply:
Nice!